src/bert_layers/model.py

```diff
@@ -506,8 +565,120 @@ def forward(
 
 
 class BertForMultipleChoice(BertPreTrainedModel):
-    # TBD: Push in future commit
-    pass
+    """
+    Bert Model with a multiple choice classification head on top (a linear layer on top of the pooled output and a
+    softmax) e.g. for RocStories/SWAG tasks.
+    """
+
+    def __init__(self, config):
+        super().__init__(config)
+        self.num_labels = config.num_labels
+        self.config = config
+
+        self.bert = BertModel(config)
+        classifier_dropout = (
+            config.classifier_dropout
+            if config.classifier_dropout is not None
+            else config.hidden_dropout_prob
+        )
+        self.dropout = nn.Dropout(classifier_dropout)
+        self.classifier = nn.Linear(config.hidden_size, 1)
```

Comment:
it took some time to me to figure out how this works (get logit for each candidate than normalize). The first thing I thought was that this should have been hidden_size --> num_labels 😓 a comment on how this type of head works would be good for posterity I think